console/src/components/IconButton/IconButton.js

```diff
@@ -0,0 +1,56 @@
+import AgateIcon from '@enact/agate/Icon';
+import Button from '@enact/agate/Button';
+import kind from '@enact/core/kind';
+import React from 'react';
+
+import componentCss from './IconButton.less';
+
+const Icon = kind({
+	name: 'Icon',
+	styles: {
+		css: componentCss,
+		className: 'icon'
+	},
+	computed: {
+		className: ({css, smallest, styler}) => {
+			return styler.append(smallest ? css.smallest : '');
+		}
+	},
+	render: (props) => {
+		delete props.smallest;
+		return (
+			<AgateIcon {...props} />
+		);
+	}
+});
+
+const IconButton = kind({
+	name: 'IconButton',
+	styles: {
+		css: componentCss
+	},
+	computed: {
+		className: ({css, smallest, styler}) => {
+			return styler.append(smallest ? css.smallest : '');
+		}
+	},
+	render: ({children, css, icon, smallest, ...rest}) => {
```

Comment:
The `children` prop is defined but not used here. I noticed I a) couldn't pass an icon as `children`, rather it needed to be specified as `icon` and b) couldn't pass a child string. Do you think we could add the ability to use `children` as the icon if `icon` is not supplied, like we do in `ui/IconButton`, otherwise passing `children` along with `icon`?